vocabularies/UI.xml

```diff
@@ -1601,6 +1601,31 @@ This is in contrast to [DataFieldForIntentBasedNavigation](#DataFieldForIntentBa
         <Annotation Term="Core.Description" String="The contents of this property must not be propagated to the app-to-app navigation context" />
       </Term>
 
-    </Schema>
+      <Term Name="UserInteraction" Type="UI.UserInteractionType" AppliesTo="NavigationProperty">
+        <Annotation Term="Core.Description" String="When the annotated navigation property occurs in a response, the UI shall use the corresponding collection to immediately interact with the user and then repeat the corresponding request with the additional information obtained from the user"/>
+        <Annotation Term="Core.LongDescription" String="A navigation property thus annotated may be interpreted as not belonging to the object model but containing auxiliary information that is not always needed."/>
+      </Term>
+      <Term Name="UserInteractionSimple" Type="Collection(Edm.PrimitiveType)" AppliesTo="Property">
+        <Annotation Term="Core.Description" String="When this instance annotation occurs in a response, the UI shall let the user choose from the collection and then repeat the corresponding request with the annotated property set to the chosen value"/>
+      </Term>
+      <ComplexType Name="UserInteractionType">
+        <Property Name="Method" Type="UI.UserInteractionMethod" Nullable="false">
+          <Annotation Term="Core.Description" String="The method for interacting with the user"/>
+        </Property>
+        <Property Name="Target" Type="Collection(Edm.PropertyPath)">
+          <Annotation Term="Core.Description" String="The properties to be set to the additional information obtained from the user"/>
+          <Annotation Term="Core.LongDescription" String="If the annotated NavigationProperty has a ReferentialConstraint, a Target need not be specified"/>

```

Comment:
Referential constraints are only allowed for single-valued navigation properties.

The description of the term says "the UI shall use the corresponding collection" - so how can there be a referential constraint?

Reply:
I'll make Target non-nullable.